core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -422,38 +423,10 @@ static function (array $uris): bool {
      */
     private function canonicalize(string $value): string
     {
-        $resolved = [];
-        $chunks = preg_split('#([\\\\/]+)#', $value, -1, PREG_SPLIT_DELIM_CAPTURE | PREG_SPLIT_NO_EMPTY);
+        $windowsPath = false !== strpos($value, '\\');
```

Comment:
Not sure if we should use this as an indicator. Why not `'\\' === \DIRECTORY_SEPARATOR`?

Reply:
Because then you would not be able to unit test it.

Reply:
We could also remove the windows handling completeley here IMO, see https://github.com/contao/contao/pull/1902#discussion_r450095692

Reply:
We are running the tests on both Unix and Windows, so this should not be problematic to test.

Reply:
No, I mean: if you depend on the `DIRECTORY_SEPARATOR`, then your unit tests have to do so as well. Otherwise you could not run a unit test with a 'Windows path' on Unix and expect it to output the right thing.

Reply:
Yes, of course, the unit test needs to use `DIRECTORY_SEPARATOR` as well when paths are not normalized.

Reply:
see https://github.com/contao/contao/pull/1902#discussion_r450233694